crowdfunding_compassion/models/crowdfunding_project.py

```diff
@@ -107,6 +107,23 @@ def _compute_number_sponsorships_reached(self):
         for project in self:
             project.number_sponsorships_reached = len(project.sponsorship_ids)
 
+    def _get_active_projects_list(self, number=999):
+        projects = self.env['crowdfunding.project'].search([
+            ("state", "!=", "xvalidation")
```

Comment:
```suggestion
            ("state", "!=", "draft")
```
state name just has been changed to draft by Stephane.